configs/system.py

```diff
@@ -9,4 +9,4 @@
 
 DISPLAY = os.getenv('DISPLAY', ':0')
 
-TEST_MODE = os.getenv('STATUS_RUNTIME_TEST_MODE')
+TEST_MODE = int(os.getenv('STATUS_RUNTIME_TEST_MODE', 0))

```

Comment:
why we convert to int? STATUS_TEST_MODE could be `1` or `True`

Reply:
We are using "0" and "1" values on CI.
If we do not convert it and the value is "0," the condition `if TEST_MODE: do_somthing()` will not work.

Reply:
> We are using "0" and "1" values on CI.

okay, fine , but it is not limiting anyone to use `true` which is acceptable and will lead to failure

Reply:
Oh, fine I will revert the changes

Reply:
reverted